code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -818,6 +823,10 @@
 			user << "<span class='warning'>You can only enthrall humans.</span>"
 			revert_cast()
 			return
+		if(iswizard(target))
+			user <<"<span class='warning'>Your tendrils discover truths in this man's mind that are forbidden even to you! Your mind recoils in horror at knowledge beyond your form before you can enslave [target].</span>"
```

Comment:
idk i feel like when you use man in the way that i have as a generalized descriptor it concedes a certain amount of respect towards an individual, that they hold a certain majesty. that it's best used in a dramatic fashion because of this heightened status of the person.

someone who makes an eldritch horror recoil in fear at the mere concepts they know is someone who fits this idea and its something i want to convey.